source/includes/api/_geostore.md

```diff
@@ -1,58 +1,71 @@
 # Geostore
 
-## What is Geostore?
+***Note — This is the documentation for version 1 of the Geostore microservice, the latest version is [version 2](https://resource-watch.github.io/doc-api/index-rw.html#geoStore-v2).***
```

Comment:
Right now the differences between the two files seem to be minimal, and AFAIK functionally they are too. Any chance we can merge the two files and DRY things?

Reply:
Removed.